eth-connector/src/lib.rs

```diff
@@ -510,19 +527,17 @@ impl Withdraw for EthConnectorContract {
 impl EngineConnectorWithdraw for EthConnectorContract {
     #[payable]
     #[result_serializer(borsh)]
+    #[pause(name = "withdraw")]
```

Comment:
It is useful that you combined the two pauses withdrawal, but unfortunately, this means that it will not be possible to pause the engine methods separately.
So I think it will be better to just remove this combining and make other engine methods pausable

Reply:
fixed: https://github.com/aurora-is-near/aurora-eth-connector/pull/71/commits/bb779f9f9cde149f957ebc7145423823755549b9